config/locales/en.yml

```diff
@@ -207,6 +207,9 @@ en:
       post_placeholder: "Help the author refine their post, or point them at other
         helpful information on the site. Mention users by @username to notify them
         of this thread by email"
+      post_placeholder_new_contributor: "%{author} is a new contributor.Take care in
+      asking for clarification,commenting and answering.Be nice and check out our
+      Code of Conduct."
```

Comment:
Could you edit the message to insert spaces after commas and periods?

Reply:
@seabl updated the pr according to the changes requested.